lib/pure/uri.nim

```diff
@@ -52,10 +52,14 @@ include includes/decode_helpers
 type
   Url* = distinct string
 
+  IpKind* = enum
```

Comment:
You unconditionally export this symbol.

Reply:
I will remove `IpKind` and use `initUri(useIpv6: bool)` for compatibility.